android-lib/src/main/java/com/kinvey/android/store/DataStore.java

```diff
@@ -757,6 +757,14 @@ public Query query() {
         return new Query(new MongoQueryFilter.MongoQueryFilterBuilder());
     }
 
+    /**
+     * This methods gets a count of entities modified locally and pending a push to the backend
+     * @return the count of sync objects for given collection
+     */
+    public long syncCount() {
+        return client.getSycManager().getCount(getCollectionName());
```

Comment:
@yuliya-guseva While we are adding this, let's also correct the spelling of the `getSycManager()` method.